assembl/configs/base_env.rc

```diff
@@ -14,6 +14,10 @@ imap_host = localhost
 piwik_host = localhost
 url_metadata_host = localhost
 changes_websocket_host = localhost
+# AWS S3 config
+aws_bucket_name =
+aws_access_key_id =
```

Comment:
Keys are for developers to connect S3 using their personal account. We shouldn't do that here.  It would be better to use an aws role, (temporary permission)  that allows hosts (like  EC2 or server) to access a s3 with their own credentials (based on IP for instance)